Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Issue.hs

```diff
@@ -181,6 +181,7 @@ issueFetchMedia _ filePath =
 
 ticketStatusCallBack :: ShortId DM.Merchant -> Common.TicketStatusCallBackReq -> Flow APISuccess
 ticketStatusCallBack _ req = do
-  _ <- QIR.findByTicketId req.ticketId >>= fromMaybeM (TicketDoesNotExist req.ticketId)
+  issue <- QIR.findByTicketId req.ticketId >>= fromMaybeM (TicketDoesNotExist req.ticketId)
   QIR.updateIssueStatus req.ticketId (toDomainIssueStatus req.status)
+  CQIR.clearIssueReportByDriverCache issue.driverId
```

Comment:
Why do we need Cached queries in IssueReport?

Reply:
They were already there I didn't add any. Just  made a fix for clearing the cache after ticket status updation.